pkg/database/migrate.go

```diff
@@ -195,6 +197,28 @@ func CheckAndMigrateDB(migrate bool, db *gorm.DB) error {
 			case "ac0b8b7c0014":
 				log.Info("Migrating database to FastTrackML schema 8073e7e037e5")
 				if err := db.Transaction(func(tx *gorm.DB) error {
+					// types for migration
+					type Base struct {
+						ID         uuid.UUID `gorm:"type:uuid;primaryKey" json:"id"`
+						CreatedAt  time.Time `json:"created_at"`
+						UpdatedAt  time.Time `json:"updated_at"`
+						IsArchived bool      `json:"-"`
+					}
+					type Dashboard struct {
+						Base
+						Name        string     `json:"name"`
+						Description string     `json:"description"`
+						AppID       *uuid.UUID `gorm:"type:uuid" json:"app_id"`
```

Comment:
Any reason for this to be a pointer?

Reply:
that's just how it was previously defined

Reply:
ah I see — that's only necessary for the JSON serialization, but let's keep it like this to avoid any confusion with the past model